imports/collections/onlineusers_private.js

```diff
@@ -0,0 +1,38 @@
+import { Meteor } from 'meteor/meteor';
+import { OnlineUsersSchema } from './onlineusers.schema';
+import moment from 'moment/moment';
+
+if (Meteor.isServer) {
+    Meteor.publish('onlineUsersForRoute', function (route) {
+        return OnlineUsersSchema.find({activeRoute: route});
+    });
+}
+
+const checkRouteParamAndAuthorization = (route, userId) => {
+    check(route, String);
+    if (!userId) {
+        throw new Meteor.Error('not-authorized');
+    }
+};
+
+Meteor.methods({
+    'onlineUsers.enterRoute'(route) {
+        const userId = Meteor.userId();
+        checkRouteParamAndAuthorization(route, userId);
+
+        OnlineUsersSchema.insert({
```

Comment:
This adds multiple documents. An alternative would be to update if this `userId` already has an entry. What's the advantage of adding new documents? Multiple sessions? But what is the result in `isUserRemotelyConnected()`? The first document available? The latest one?